games-strategy/warmux/warmux-11.04.1-r2.ebuild

```diff
@@ -13,24 +13,28 @@ S="${WORKDIR}"/${PN}-11.04
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
-IUSE="debug nls unicode"
+IUSE="debug unicode"
 
 RDEPEND="
 	dev-libs/libxml2
+	media-libs/libpng:=
 	media-libs/libsdl[joystick,video,X]
 	media-libs/sdl-image[jpeg,png]
 	media-libs/sdl-mixer[vorbis]
 	media-libs/sdl-ttf
 	media-libs/sdl-net
-	media-libs/sdl-gfx
+	media-libs/sdl-gfx:=
 	media-fonts/dejavu
 	net-misc/curl
 	x11-libs/libX11
-	nls? ( virtual/libintl )
-	unicode? ( dev-libs/fribidi )"
-DEPEND="${RDEPEND}
+	virtual/libintl
+	unicode? ( dev-libs/fribidi )
+"
+DEPEND="${RDEPEND}"
```

Comment:
Missing `x11-base/xorg-proto`

Reply:
Only build/android/jni needs it?

Reply:
sdl[X] needs it, try to force remove xorg-proto and then emerge (which won't pull xorg-proto because it's a not a dep)